Explicitly disable building of _tkinter module
Simon Branford (University of Birmingham)
--- configure.orig	2024-05-24 11:09:00.888859445 +0100
+++ configure	2024-05-24 11:52:11.840124559 +0100
@@ -30585,11 +30585,11 @@
   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for stdlib extension module _tkinter" >&5
 printf %s "checking for stdlib extension module _tkinter... " >&6; }
         if test "$py_cv_module__tkinter" != "n/a"
 then :
 
-    if true
+    if false
 then :
   if test "$have_tcltk" = "yes"
 then :
   py_cv_module__tkinter=yes
 else $as_nop
